doc/sphinx-guides/source/installation/config.rst

```diff
@@ -942,6 +942,12 @@ The DPN archiver defines three custom settings, one of which is required (the ot
 
 :DuraCloudPort and :DuraCloudContext are also defined if you are not using the defaults ("443" and "duracloud" respectively). (Note\: these settings are only in effect if they are listed in the \:ArchiverSettings. Otherwise, they will not be passed to the DuraCloud Archiver class.)
 
+It also can use one setting that is common to all Archivers: :BagGeneratorThreads
+
+``curl http://localhost:8080/api/admin/settings/:BagGenerator -X PUT -d '8'``
```

Comment:
This should be `:BagGeneratorThreads` instead of `:BagGenerator` in the curl example, right?

Reply:
Seems to be fixed. Thanks.